doc/source/contributing.rst

```diff
@@ -207,6 +206,29 @@ See the full conda docs `here <http://conda.pydata.org/docs>`__.
 
 At this point you can easily do an *in-place* install, as detailed in the next section.
 
+.. _contributing.windows:
+
+Creating a Windows development environment
+------------------------------------------
+
+To build on windows, you need to have compilers installed to build the extensions. You will need to install the appropriate Visual Studio compilers, VS 2008 for Python 2.7, VS 2010 for 3.4, and VS 2015 for Python 3.5.
+
+For Python 2.7, you can install the ``mingw`` compiler which will work equivalently to VS2008::
```

Comment:
What do you mean by minimal example?  Unfortunately, I myself have had issues downloading `mingw-64` because my machine already has `mingw` installed, but from what I have seen in the `numpy` discussions, that seems to be the way to go now.  If we want more feedback on this, perhaps we could grab someone from there to provide input?